MonoGame.Framework/Audio/SoundEffect.cs

```diff
@@ -262,13 +262,7 @@ public static int GetSampleSizeInBytes(TimeSpan duration, int sampleRate, AudioC
         /// </remarks>
         public bool Play()
         {
-            var inst = GetPooledInstance(false);
-            if (inst == null)
-                return false;
-
-            inst.Play();
-
-            return true;
+            return Play(1f, 0f, 0f);

```

Comment:
Good point. I took a look at this, but I think to do this well it would require a bit more work than what I have time for at the moment. For example, WinDX calls `PlatformSetupInstance` after getting the instance. This can potentially re-create the voice, which means any non-default settings must be applied after it's called. So it works now, but if I changed it to reset the values in the pool it could get reset by `PlatformSetupInstance` afterwards.

So I'll just close this PR for now, and I may look into improving this area of the code at a later point.